src/libaktualizr/primary/aktualizr_test.cc

```diff
@@ -635,7 +635,7 @@ TEST(Aktualizr, FinalizationFailure) {
     EXPECT_EQ(current_version, SIZE_MAX);
 
     auto aktualizr_cycle_thread = aktualizr.RunForever();
-    auto aktualizr_cycle_thread_status = aktualizr_cycle_thread.wait_for(std::chrono::seconds(10));
+    auto aktualizr_cycle_thread_status = aktualizr_cycle_thread.wait_for(std::chrono::seconds(20));
```

Comment:
:( Is there a way we can get rid of the explicit times? I guess 20 seconds isn't so bad, but still.

Reply:
Hm I don't really know how to do that better. Waiting forever and relying on something else to timeout would probably lead to worse error messages.

Putting less load on CI machines is the best bet I would say and it's ongoing.

Reply:
No worries, it's fine. I just got triggered by the hardcoded numbers, you know, but they aren't always evil.